ui/core/components/results_viewer.tsx

```diff
@@ -127,4 +133,20 @@ export class ResultsViewer extends Component {
 		this.contentElem.style.display = 'block';
 		this.pendingElem.style.display = 'none';
 	}
+
+	addAbortButton(abortClicked: (event: MouseEvent) => void) {
+		this.buttonElem.innerHTML = '';
+		this.buttonElem.style.display = 'block';
+		this.buttonElem.appendChild(
```

Comment:
Remove `this.buttonElem.innerHTML = '';` and then change this to `this.buttonElem.replaceChildren()`. Actually the more "performant" due to only triggering 1 render.

Reply:
Replaced it.